provider/go.mod

```diff
@@ -13,8 +13,8 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.4.3
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.46.0
-	github.com/pulumi/pulumi/sdk/v3 v3.46.0
+	github.com/pulumi/pulumi/pkg/v3 v3.45.1-0.20221108173417-4910d9cc9a88
+	github.com/pulumi/pulumi/sdk/v3 v3.46.1-0.20221108173417-4910d9cc9a88
```

Comment:
Hmm what's with the different leading semver between the two but the trailing snapshot version?

Reply:
It looks like the pulumi/pulumi `pkg` and `sdk` tags for `v3.46.0` are on different commits, so I think that's why.